src/audio/CMakeLists.txt

```diff
@@ -169,7 +169,8 @@ check_optimization(fma -mfma -ftree-vectorize -DOPS_FMA)
 check_optimization(hifi2ep -mhifi2ep "" -DOPS_HIFI2EP)
 check_optimization(hifi3 -mhifi3 "" -DOPS_HIFI3)
 
-set(sof_audio_modules mixer volume src asrc eq-fir eq-iir dcblock crossover tdfb drc multiband_drc mfcc mux)
+set(sof_audio_modules mixer volume src asrc eq-fir eq-iir dcblock crossover
+    tdfb drc multiband_drc mfcc mux dts)
```

Comment:
May I know why is this commit required?
If yes, may it be moved to src/audio/module_adapter/CMakeLists.txt?

Reply:
I think it could be removed now as it's already handled in src/audio/module_adapter/CMakeLists.txt.